parquet/src/test/java/org/apache/iceberg/parquet/TestParquetSchemaUtil.java

```diff
@@ -464,4 +537,20 @@ private Type map(Integer id, String name, Repetition repetition, Type keyType, T
     }
     return builder.named(name);
   }
+
+  private Type variant(Integer id, String name, Repetition repetition) {
+    GroupBuilder<GroupType> builder =
+        org.apache.parquet.schema.Types.buildGroup(repetition)
+            .as(
+                org.apache.parquet.schema.LogicalTypeAnnotation.variantType(
```

Comment:
nit: import `org.apache.parquet.schema.LogicalTypeAnnotation` so we don't need the fully qualified name here.